spec/player_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/player'
+require './lib/board'
+
+describe Player do
+  it 'switch player 1' do
+    player = Player.new
+    player.switch_player
+    expect(player.player).to eq 1
+    expect(player.symbol).to eq 'X'
+  end
+
+  it 'switch the player 2' do
+    player = Player.new
+    player.switch_player
+    player.switch_player
+    expect(player.player).to eq 2
+    expect(player.symbol).to eq '●'
+  end
+
+  it 'player makes a valid move' do
+    player = Player.new
+    board = Board.new
+    expect(player.move(1, board)).to eq true
+  end
+
+  it 'player makes a invalid move' do
+    player = Player.new
+    board = Board.new
+    expect(player.move(10, board)).to eq false
+  end
```

Comment:
- [ ] Currently, you have all methods for each class wrapped in one `describe` block and which should not be. You should ensure that the specs for each method be wrapped in one `describe` block. 
For example in the highlighted code above, it should be wrapped in its own `describe` block, with one `it` block for each expected behavior just like you have it and the behaviors should test for positive and/or negative cases.

#### **Kindly note that this does NOT ONLY apply to the highlighted code, ensure you make these changes in all three test files as it appears that the structure for defining a `describe` block was not exactly followed.**